handler/handler.go

```diff
@@ -1,396 +0,0 @@
-package handler
```

Comment:
This file is too large, may consider de-couple into multiple files for different endpoints.